src/components/TransactionStatusToast/types.ts

```diff
@@ -0,0 +1 @@
+export type StatusIconType = 'info' | 'error' | 'warning';
```

Comment:
I think that an Enum would be more suitable because you can then do something like `MyEnum.INFO` when creating switch case declarations